testdata/scripts/extract.txt

```diff
@@ -545,6 +548,33 @@ Provides CRUD operations on the accounts resource.
 
 Creates a new account
 
+```sh
+curl -X POST \
+    https://openbank.com/v1/accounts \
+    -H 'Authorization: Bearer USE_YOUR_TOKEN' \
+    -d '{
+	"account_id": "string",
```

Comment:
I think indenting the JSON with a leading two tabs would also improve readability here.